docs/getting-started-guides/sysdig.md

```diff
@@ -0,0 +1,118 @@
+---
+---
+
+Kubernetes comes built-in with some basic resource utilization monitoring as described at the [Monitoring](/docs/user-guide/monitoring) user guide page. Here we describe how to utilize distributed, full-stack application monitoring and deep system troubleshooting for your Kubernetes cluster with Sysdig.
+
+### Kubernetes and Sysdig
+Sysdig offers native, first class support for Kubernetes. That includes [sysdig](http://www.sysdig.org/), an open source system troubleshooting tool, and [Sysdig Cloud](https://sysdig.com/), a monitoring solution that has been designed from the ground up to support containers and microservices.
+
+In this post we will quickly preview the Kubernetes visibility in both open source sysdig and Sysdig Cloud, covering two primary use cases:
```

Comment:
Slightly confusing sentence.  I'd suggest replacing "In this post we will quickly preview the Kubernetes visibility in both open source sysdig and Sysdig Cloud, covering two primary use cases:" with
"This document covers two primary use cases for using open source sysdig and Sysdig Cloud to provide visibility into your Kubernetes deployments:"